### PR TITLE
Add link to /test/overview from a parent job group

### PR DESCRIPTION
**Pull request by ilausuch**
Provides link to /tests/overview of latest builds of all job groups
within a parent job group

https://progress.opensuse.org/issues/94732

### CODE REVIEW

File: t/22-dashboard.t [hunk 1]
```diff
@@ -23,7 +23,7 @@ use Mojo::Base -signatures;
 use Test::Mojo;
 use Test::Warnings ':report_warnings';
 use OpenQA::Test::Case;
-use OpenQA::Test::TimeLimit '10';
+use OpenQA::Test::TimeLimit '50';
```
Comment (Martchus) [inline t/22-dashboard.t:26]: This is a high increase. Is this really required?

File: t/22-dashboard.t [hunk 2]
```diff
@@ -194,6 +194,7 @@ for my $url (@urls) {
 # parent group overview
 $t->get_ok('/parent_group_overview/' . $test_parent->id)->status_is(200);
 check_test_parent('expanded');
+$t->element_exists('#test_result_overview_link');
```
Comment (Martchus) [inline t/22-dashboard.t:197]: Of course we should check whether the link is actually correct.

File: templates/webapi/main/parent_group_overview.html.ep [hunk 3]
```diff
@@ -1,3 +1,4 @@
+% use Data::Dumper qw(Dumper);
```
Comment (Martchus) [inline templates/webapi/main/parent_group_overview.html.ep:1]: Should be removed later

File: templates/webapi/main/parent_group_overview.html.ep [hunk 4]
```diff
@@ -21,6 +22,15 @@
     Last Builds for <%= $group->{name} %>
 </h2>
 
+<p>
+  <a href="<%= url_for('tests_overview')->query((todo => 1), map { (groupid => $_->{id}) } @{$children});%>"
```
Comment (Martchus) [inline templates/webapi/main/parent_group_overview.html.ep:26]:
```suggestion
  <a href="<%= url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{id}) } @$children); %>"
```

File: templates/webapi/main/parent_group_overview.html.ep [hunk 5]
```diff
@@ -21,6 +22,15 @@
...
+    title="Shows the latest builds of all job groups within this parent"
+    id="test_result_overview_link"
+  >
+    Test result overview
+  </a>
+<p>
```
Comment (Martchus) [inline templates/webapi/main/parent_group_overview.html.ep:32]:
```suggestion
</p>
```

**Review by Martchus — CHANGES_REQUESTED** [review]

File: t/22-dashboard.t (hunk 1 above)
Comment (ilausuch) [reply t/22-dashboard.t:26]: I know, please this is still WIP

File: t/22-dashboard.t (hunk 2 above)
Comment (ilausuch) [reply t/22-dashboard.t:197]: Yes, this is under development yet

File: templates/webapi/main/parent_group_overview.html.ep [hunk 6]
```diff
@@ -21,6 +22,15 @@
     Last Builds for <%= $group->{name} %>
 </h2>
 
+<p>
+  <a href="<%= url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{id}) } @$children);%>"
+    title="Shows the latest builds of all job groups within this parent"
```
Comment (okurz) [inline templates/webapi/main/parent_group_overview.html.ep:27]:
```suggestion
    title="Shows the latest test results for all job groups within this parent job group"
```

**Review by okurz — CHANGES_REQUESTED** [review]

File: templates/webapi/main/parent_group_overview.html.ep [hunk 7]
```diff
@@ -21,6 +21,15 @@
     Last Builds for <%= $group->{name} %>
 </h2>
 
+<p>
+  <a href="<%= url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{id}) } @$children);%>"
```
Comment (Martchus) [inline templates/webapi/main/parent_group_overview.html.ep:25]:
```suggestion
  <a href="<%= url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{id}) } @$children); %>"
```
(The space after the `;` is still missing. Note that tidy doesn't run on templates so you'll have to take care of this yourself.)

**Review by Martchus — CHANGES_REQUESTED** [review]

**Review by okurz — APPROVED** [review]

File: templates/webapi/main/parent_group_overview.html.ep (hunk 7 above)
Comment (ilausuch) [reply templates/webapi/main/parent_group_overview.html.ep]: Thanks, solved

**Review by perlpunk — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Pull request merged**